tests/unit/interpreter/CMakeLists.txt

```diff
@@ -36,4 +36,12 @@ add_executable (interpreter_test ${unit_test_sources})
 
 target_link_libraries (interpreter_test gtest_main )
 
+# Copy WASM files to build directory
```

Comment:
Preferably store all those test cases in a separate directory, such as smart-tests or digit-suite. 

please co-sign with ai-assistant in the git comment. https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors#creating-co-authored-commits-on-the-command-line